frontend/.env

```diff
@@ -0,0 +1 @@
+SERVER_URL=10.72.161.24
```

Comment:
.env file should be ignored

Reply:
dotenv is merged, you can use it now @GuJialu